src/main/kotlin/de/timo_reymann/mjml_support/tagprovider/BuiltInMjmlTagInformationProvider.kt

```diff
@@ -690,8 +702,16 @@ class BuiltInMjmlTagInformationProvider : MjmlTagInformationProvider() {
                     ),
                     ATTRIBUTE_ALT,
                     ATTRIBUTE_BORDER,
+                    ATTRIBUTE_BORDER_TOP,
+                    ATTRIBUTE_BORDER_BOTTOM,
+                    ATTRIBUTE_BORDER_LEFT,
+                    ATTRIBUTE_BORDER_RIGHT,
                     ATTRIBUTE_BORDER_RADIUS,
-                    ATTRIBUTE_CONTAINER_BACKGROUND_COLOR,
+                    MjmlAttributeInformation(
+                        "container-background-color",
+                        MjmlAttributeType.COLOR,
+                        "inner element background color"
+                    ),
```

Comment:
chore: Please refactor the container-background-color attributes into a constant like you did for `ATTRIBUTE_BACKGROUND_POSITION_*`

Reply:
First off, thank you very much for the awesome plugin!

I appreciate you taking the time to consider this pull request. Initially, I was going to add the new constant, but there was already a constant found in `src/main/kotlin/de/timo_reymann/mjml_support/model/constants_attributes.kt`

That constant attribute had a different description, so I decided not to change the existing constant attribute for `container-background-color` and generate the attribute using the MjmlAttributeInformation() method.

Here is the snippet that exists in `src/main/kotlin/de/timo_reymann/mjml_support/model/constants_attributes.kt` on line 206.

```
val ATTRIBUTE_CONTAINER_BACKGROUND_COLOR = MjmlAttributeInformation(
    "container-background-color",
    MjmlAttributeType.COLOR,
    "background color of the cell"
)
```

Would you like me to create a new attribute (something such as `ATTRIBUTE_CONTAINER_BACKGROUND_COLOR_INNER`) and replace the MjmlAttributeInformation() methods I have added with that new constant? You give the direction and I will be happy to implement.

Reply:
Ah I see, than its fine to keep it :)